gatsby-node.js

```diff
@@ -1,4 +1,90 @@
+const Sentry = require("@sentry/node");
+require("@sentry/tracing");
+const Profiling = require("@sentry/profiling-node");
+
+const activeEnv =
+  process.env.GATSBY_ENV || process.env.NODE_ENV || "development";
+
+Sentry.init({
+  dsn:
+    process.env.SENTRY_DSN ||
+    "https://f107f3f0deb544289e4e056922e5e5a4@o1.ingest.sentry.io/5266138",
+  debug: true,
+  environment: activeEnv,
+  tracesSampleRate: activeEnv === "development" ? 0 : 1,
+  profilesSampleRate: activeEnv === "development" ? 0 : 1, // Set profiling sampling rate.
+  integrations: [new Profiling.ProfilingIntegration()],
+});
+
 exports.onCreateNode = require("./src/gatsby/onCreateNode").default;
 exports.onCreateWebpackConfig = require("./src/gatsby/onCreateWebpackConfig").default;
 exports.createPages = require("./src/gatsby/createPages").default;
 exports.createSchemaCustomization = require("./src/gatsby/createSchemaCustomization").default;
+
+Sentry.configureScope(scope => {
+  const id = process.env.GITHUB_RUN_ID;
+  if (id) {
+    scope.setUser({ id });
+  }
+});
+
+function startFromPrevTransaction(newTransactionContext) {
```

Comment:
the api making me have to add this helper

![scream-toad](https://user-images.githubusercontent.com/18689448/196664572-4d26d0bb-83a6-474a-8fa2-135064467349.gif)